vlib/v/tests/known_errors/testdata/cast_aliases_on_generics_type.vv

```diff
@@ -26,7 +24,7 @@ fn test_report() {
 	r := Opt<ParseRes>(None<ParseRes>{})
 	match r {
 		Some<ParseRes> {
-			rx := Result<[]Token, ParseErr>(r)
+			rx := Result<[]Token, ParseErr>(r.value)
 		}
 		None<ParseRes> {}
 	}
```

Comment:
the changes here seem unrelated to the PR ?

Reply:
>the changes here seem unrelated to the PR ?

Yes, I originally copy and paste the problem from the issue, but the issue contains the wrong code, r inside the match will begin with type `Some<T>` and not `ParseRes`.

This was confusing to me originally but after a while, I got it with this fix :)